man/systemd.network.xml

```diff
@@ -1903,6 +1912,19 @@
         <varname>DNS=</varname>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>EmitSMTPServer=</varname></term>
```

Comment:
Same as in the other PR: we can assume that the user wants to emit the ifnormation if SMTPServer= is configured.